### ISSUE TITLE
empty matrix in get_laplacian.m

### ISSUE THREAD

**Issue by andreapago** [sjtrny/Dark-Channel-Haze-Removal#2]
Hi @sjtrny 
I was looking into get Laplacian and I was testing it with a very simple image (resize of the original forest image to 10% of the original size). I see that the `win_inds` array becomes:

```
Empty matrix: 1-by-0
```

since `n_min` becomes bigger than `n_max`.

(When `k` is around 8010 just to give you a reference.)
The emptiness of the matrix has problems that hit then `num_neigh` that becomes 0 and `win_var` becomes `NaN`.
Is this behavior on purpose? I quite don't get it. Can you, please, give me some details on this? Since of course R, to which I am porting the code and idea, complains much more than matlab already when the indexes of the `win_inds` don't match anymore the dimensions.

Many thanks in advance.

**Comment by sjtrny** [comment]
No it's not intentional behaviour. I borrowed get_laplacian from some code I used for alpha matting (also known as image matting). So that's why the reference to "trimap" was there. The trimap marks which pixels one needs to estimate the alpha for so we limit laplacian construction to only those pixels to save time. But in this case since there is no trimap the values in "indices" was incorrect and then the values "i" and "j" returned by ind2sub was also incorrect, which meant that sometimes n_min > n_max.

Anyway this is all to say that I removed thel references to the trimap functionality. Let me know if it's still happening.
